### PR TITLE
[Admin] Add user and server info tooling

### PR DESCRIPTION
**Pull request by flaree**

### CODE REVIEW

File: ballsdex/packages/admin/cog.py [hunk 1]
```diff
@@ -1391,3 +1393,126 @@ async def trade_info(
...
+            spawn_enabled = False
+
+        total_server_balls = await BallInstance.filter(
+            catch_date__gte=datetime.datetime.now() - datetime.timedelta(days=days),
+            server_id=guild.id,
+        ).prefetch_related("player")
+
+        embed = discord.Embed(
+            title=f"{guild.name} ({guild.id})",
+            description=f"Owner: {guild.owner} ({guild.owner_id})",
```
Comment (ltzmax) [inline ballsdex/packages/admin/cog.py:1445]: `guild.owner` is showing as None here, but i assume its about caching?

File: ballsdex/packages/admin/cog.py [hunk 2]
```diff
@@ -1391,3 +1393,126 @@ async def trade_info(
...
+        )
+        embed.add_field(
+            name=f"Total Unique {settings.collectible_name}s Caught",
+            value=len(set([x.countryball for x in total_user_balls])),
+        )
+        embed.add_field(
+            name=f"Total Server with {settings.collectible_name}s Caught",
+            value=len(set([x.server_id for x in total_user_balls])),
+        )
+        embed.set_thumbnail(url=user.avatar.url)  # type: ignore
```
[the remark this answers is not in the document]
Comment (ltzmax) [reply ballsdex/packages/admin/cog.py:1517]:
```suggestion
        embed.set_thumbnail(url=user.display_avatar)  # type: ignore
```

File: ballsdex/packages/admin/cog.py (hunk 1 above)
Comment (flaree) [reply ballsdex/packages/admin/cog.py:1445]: ah yes, it'll have to be changed to fetch_user on the owner is. Do you mind suggesting that?

Comment (ltzmax) [inline ballsdex/packages/admin/cog.py:1445]:
```suggestion
        owner = await self.bot.fetch_user(guild.owner_id)
        embed = discord.Embed(
            title=f"{guild.name} ({guild.id})",
            description=f"Owner: {owner} ({guild.owner_id})",
```

**Review by ltzmax — APPROVED** [review]

**Pull request merged**